notebooks/debug_pipeline.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
The notebook is good to have but would it be possible to offer the information that is in here in the documentation as well? If that is work for a later commit that is fine, we can always add it later, but typically the docs should be the central place where to demo code.

Reply:
Done.